config/schema.yaml

```diff
@@ -0,0 +1,340 @@
+          SCD_THREADS: 2
+SCD_MAX_EXECUTION_TIME:
+  description: Allows you to increase the maximum expected execution time for the static content deploy command.
+    By default, Magento Commerce sets the maximum expected execution to 400 seconds, but in some scenarios you might
+    need more time to complete the static content deployment for a Cloud project.
+  type: integer
+  magento_version: '>=2.2.0'
+  stages:
+    - global
+    - build
+    - deploy
+  default:
+    build: null
+    deploy: null
+  examples:
+    - stage:
+        global:
+          SCD_MAX_EXECUTION_TIME: 3600
+SCD_MATRIX:
+  description: Allows you to configure multiple locales per theme as long as the theme is not excluded using
+    the SCD_EXCLUDE_THEMES variable during deployment. This is ideal if you want to speed up thedeployment process
+    by reducing the amount of unnecessary theme files. For example, you can deploy the magento/backend theme in
+    English and a custom theme in other languages.
+  type: array
+  stages:
+    - global
+    - build
+    - deploy
+  default:
+    build: {}
+    deploy: {}
+  examples:
+    - comment: The following example deploys the magento/backend theme with three locales
+      stage:
+        deploy:
+          SCD_MATRIX:
+            "magento/backend":
+              language:
+                - en_US
+                - fr_FR
+                - af_ZA
+    - comment: Also, you can choose to not deploy a theme
+      stage:
+        deploy:
+          SCD_MATRIX:
+            "magento/backend": []
+SKIP_SCD:
+  description: Skips static content deployment during the build/deploy phase.
+  type: boolean
+  stages:
+    - global
+    - build
+    - deploy
+  default:
+    build: false
+    deploy: false
+  examples:
+    - stage:
+        build:
+          SKIP_SCD: true
+SKIP_HTML_MINIFICATION:
+  description: Skip copying the static view files in the var/view_preprocessed directory
+    to reduce downtime when deploying to the Staging and Production environments
+    and generates minified HTML when requested.
+  type: boolean
+  stages:
+    - global
+  default:
+    global: true
+  examples:
+    - stage:
+        global:
+          SKIP_HTML_MINIFICATION: true
+SCD_ON_DEMAND:
+  description: Enable generation of static content when requested by a user.
+    Pre-loading the cache using the post_deploy hook reduces site downtime.
+  type: boolean
+  stages:
+    - global
+  default:
+    global: false
+  examples:
+    stage:
+      global:
+        SCD_ON_DEMAND: true
+MIN_LOGGING_LEVEL:
+  description: Use to override the minimum logging level for all output streams without making changes to the code.
+    This helps to improve troubleshooting problems with deployment. For example, if your deployment fails,
+    you can use this variable to increase the logging granularity globally. Doesn't affect log level for the file output.
+    See Set up notifications—Log levels.
+  type: string
+  allowed:
+    - ''
```

Comment:
Do we need to allow an empty value? If an option is added to .magento.env.yaml it shouldn't be empty.

Reply:
Makes sense, fixed

Reply:
But actually you will lose possibility to overwrite it per stage